genie-agent/job-execution-state-machine.dot

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+digraph JobExecution {
```

Comment:
Unfortunately not, we could write a command/script that exports this from the state machine bean (it's relatively simple since our statemachine is simple and does not use advanced features such as hierarchical states, fork, joins, etc).
Until now, I've updated the graph first, and then used it as a map to make the corresponding changes in config.